libraries/Updater/library.properties

```diff
@@ -0,0 +1,10 @@
+name=Updater
+version=1.0
+author=
```

Comment:
From what I understand, it's @me-no-dev 
author=me-no-dev <hristo@espressif.com>

https://github.com/esp8266/Arduino/commits/master?after=a8873c2364612cd1f8225cd038cc6d58a482bfd8+34&path%5B%5D=cores&path%5B%5D=esp8266&path%5B%5D=Updater.cpp